lib/dradis/plugins/projects/upload/v3/template.rb

```diff
@@ -70,12 +70,11 @@ def finalize_lists
       # Private: Restore Board, List and Card information from the project
       # template.
       def parse_methodologies(template)
-        if template_version == 1
-          # Restore Board from old xml methodology format
-          process_v1_methodologies(template)
-        else
-          process_v2_methodologies(template)
-        end
+        # old methodologies format
+        process_v1_methodologies(template)
+
+        # new methodologies format
+        process_v2_methodologies(template)
```

Comment:
In the event a methodology is imported by `process_v1_methodologies(template)` does `process_v2_methodologies(template)` essentially become a no-op?
Or vice-versa. It'll run but nothing will happen?

Reply:
Exactly, that is the idea.
The code is already there:
If no old style methodologies are present, we just return: https://github.com/dradis/dradis-projects/blob/master/lib/dradis/plugins/projects/upload/v3/template.rb#L169
In no new methodologies are present, we don't enter the loop:
https://github.com/dradis/dradis-projects/blob/master/lib/dradis/plugins/projects/upload/v3/template.rb#L130
I though on adding an explicit `return` in new methodologies too. But I think it works like this.